src/code42cli/cmds/alerts.py

```diff
@@ -16,18 +15,26 @@
 from code42cli.click_ext.groups import OrderedGroup
 from code42cli.cmds.search import SendToCommand
 from code42cli.cmds.search.cursor_store import AlertCursorStore
-from code42cli.cmds.search.extraction import handle_no_events
 from code42cli.cmds.search.options import server_options
+from code42cli.cmds.util import convert_to_or_query
+from code42cli.cmds.util import create_time_range_filter
+from code42cli.cmds.util import try_get_default_header
+from code42cli.cmds.util import verify_filter_groups
 from code42cli.date_helper import convert_datetime_to_timestamp
 from code42cli.date_helper import limit_date_range
 from code42cli.file_readers import read_csv_arg
 from code42cli.options import format_option
 from code42cli.output_formats import JsonOutputFormat
 from code42cli.output_formats import OutputFormat
 from code42cli.output_formats import OutputFormatter
-
+from code42cli.util import hash_event
+from code42cli.util import parse_timestamp
+from code42cli.util import warn_interrupt
 
 ALERTS_KEYWORD = "alerts"
+MAX_ALERT_PAGE_SIZE = 500
+_ALERT_DETAIL_BATCH_SIZE = 100
```

Comment:
Could resolve NTEG-1917 while we're at it here.

Reply:
I think just the detail batch, as that's where they were seeing performance issues.